server/src/main/java/org/candlepin/guice/CandlepinContextListener.java

```diff
@@ -382,4 +401,15 @@ private void insertValidationEventListeners(Injector injector) {
         registry.getEventListenerGroup(EventType.PRE_UPDATE).appendListener(listenerProvider.get());
         registry.getEventListenerGroup(EventType.PRE_DELETE).appendListener(listenerProvider.get());
     }
+
+
+    private String getCrlFilePath() {
+        String filePath = config.getString(ConfigProperties.CRL_FILE_PATH);
+
+        if (filePath == null) {
+            throw new IseException("CRL file path not defined in config file");
```

Comment:
IseException is a REST exception. We should throw something like a ConfigurationException here instead.

Reply:
Done.